codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/http/HttpBindingGenerator.kt

```diff
@@ -157,7 +157,7 @@ class HttpBindingGenerator(
                     let headers = #T::headers_for_prefix(header_map, ${binding.locationName.dq()});
                     let out: std::result::Result<_, _> = headers.map(|(key, header_name)| {
                         let values = header_map.get_all(header_name);
-                        #T(values.iter()).map(|v| (key.to_string(), v.unwrap()))
+                        #T(values.iter()).map(|v| (key.to_string(), v.expect("we've checked there's at least one value for this header name")))
```

Comment:
```suggestion
                        #T(values.iter()).map(|v| (key.to_string(), v.expect("We have checked there is at least one value for this header name")))
```

Reply:
I don't know if there are API guidelines for panic error messages, but I start all of these with lowercase as per https://doc.rust-lang.org/nightly/std/error/trait.Error.html:

> Error messages are typically concise lowercase sentences without trailing punctuation:

I'd only include punctuation (full-stops / starting with uppercase) in multi-sentence error messages.

---

I've removed the contractions though.